test/test.js

```diff
@@ -581,15 +581,15 @@ describe('parser (exif data)', () => {
 				assert.equal(exif.YResolution, 72)
 				assert.equal(exif.ResolutionUnit, 2)
 				assert.equal(exif.Software, 'V06.02.20')
```

Comment:
All the tests are now the same as exiftool. It will allow me to use `moment.utc(...)` on frontend.